package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0-alpha.2",
   "description": "The JS Buy SDK is a lightweight library that allows you to build ecommerce into any website. It is based on Shopify's API and provides the ability to retrieve products and collections from your shop, add products to a cart, and checkout.",
   "main": "index.js",
-  "jsnext:main": "index.es.js",
+  "module": "index.es.js",
```

Comment:
maybe backport this to the other rollup-ed modules as well? `graphql-js-client`, and `graphql-js-schema`?